pkgs/tools/archivers/maxcso/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libuv, lz4, zlib }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  pname = "maxcso";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "unknownbrackets";
+    repo = "maxcso";
+    rev = "v${version}";
+    sha256 = "10r0vb3ndpq1pw5224d48nim5xz8jj94zhlfy29br6h6jblq8zap";
+  };
+
+  buildInputs = [ libuv lz4 zlib ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = {
+    homepage = "https://github.com/unknownbrackets/maxcso/";
```

Let's drop the tailing slash:
```suggestion
    homepage = "https://github.com/unknownbrackets/maxcso";
```